README.md

```diff
@@ -48,75 +48,17 @@ Using Maven:
-        // indexesHealth.resetStatistics();
-    }
-
-    private static void forProduction() {
-        final String writeUrl = "jdbc:postgresql://host-name-1:6432,host-name-2:6432,host-name-3:6432/db_name_production?ssl=true&targetServerType=master&prepareThreshold=0&preparedStatementCacheQueries=0&connectTimeout=2&socketTimeout=50&loginTimeout=10&sslmode=require";
-        final String readUrl = "jdbc:postgresql://host-name-1:6432,host-name-2:6432,host-name-3:6432,host-name-4:6432,host-name-5:6432/db_name_production?ssl=true&targetServerType=preferSlave&loadBalanceHosts=true&prepareThreshold=0&preparedStatementCacheQueries=0&connectTimeout=2&socketTimeout=50&loginTimeout=10&sslmode=require";
-        final String cascadeAsyncReadUrl = "jdbc:postgresql://host-name-6:6432/db_name_production?ssl=true&targetServerType=preferSlave&loadBalanceHosts=true&prepareThreshold=0&preparedStatementCacheQueries=0&connectTimeout=2&socketTimeout=50&loginTimeout=10&sslmode=require";
-        final String userName = "user_name_production";
-        final String password = "password_production";
-        final HighAvailabilityPgConnectionFactory haPgConnectionFactory = new HighAvailabilityPgConnectionFactoryImpl(new PgConnectionFactoryImpl());
-        final HighAvailabilityPgConnection haPgConnection = haPgConnectionFactory.of(writeUrl, userName, password, readUrl, cascadeAsyncReadUrl);
-        final IndexesHealth indexesHealth = new IndexesHealthImpl(haPgConnection, new MaintenanceFactoryImpl());
-        final Exclusions exclusions = Exclusions.builder()
-                .withIndexSizeThreshold(10, MemoryUnit.MB)
-                .withTableSizeThreshold(10, MemoryUnit.MB)
-                .build();
-        final IndexesHealthLogger logger = new SimpleHealthLogger(indexesHealth);
-        logger.logAll(exclusions, PgContext.ofPublic())
-                .forEach(System.out::println);
-    }
-}
-```
+## How to use
+There are three main scenarios of using **pg-index-health** in your projects:
+* unit\functional testing;
+* collecting indexes health data and monitoring bloat;
+* analysis of database configuration.
 
-For more examples see [pg-index-health-demo](https://github.com/mfvanek/pg-index-health-demo) project.
+All these cases are covered with examples in the [pg-index-health-demo](https://github.com/mfvanek/pg-index-health-demo) project.
 
 ## Questions, issues, feature requests and contributions
+* If you have any question or a problem with the library, please [file an issue](https://github.com/mfvanek/pg-index-health/issues).
+* Contributions are always welcome! Please see [contributing guide](CONTRIBUTING.md) for more details.
 
- * If you have any question or a problem with the library, please [file an issue](https://github.com/mfvanek/pg-index-health/issues).
- * Contributions are always welcome! Please see [contributing guide](CONTRIBUTING.md) for more details.
+## Related projects
+[pg_analyse](https://github.com/idlesign/pg_analyse)
```

Comment:
I think it would be better to add some information about the project.

Reply:
fixed